### ISSUE TITLE
Staging Review finding: Dynamic alerts not announced to screen readers

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#123442]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** MHV Horizon
**Product:** MHV on VA.gov
**Feature:** Notification email update prompts

## Findings details
**VA.gov Experience Standard - issue:** [User can't perceive an element.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

Dynamic alerts are not announced to screen reader users, meaning these users are not aware of the current status and may miss the success and error states. 

<details><summary>Link, screenshot or steps to recreate</summary>

Success alert not announced:
<img width="1394" height="882" alt="success not announced" src="https://github.com/user-attachments/assets/1732adbc-8597-414f-ad5c-7c28cbac589e" />

Error alert not announced:
<img width="1408" height="1310" alt="Error not announced" src="https://github.com/user-attachments/assets/b734b2aa-00a0-42fe-836d-8c6f72752366" />

</details>

## Recommended action

Ensure dynamic error messages are announced to screen reader users. 

https://design.va.gov/components/alert/#assign-an-appropriate-aria-role

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 4.1.3 Status Messages - Level AA](https://www.w3.org/TR/WCAG22/#status-messages)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
